.travis.yml

```diff
@@ -47,10 +53,11 @@ before_install:
 install:
   - travis_retry pip install --upgrade pip
   - travis_retry pip install coveralls
-  - travis_retry pip install pytest pytest-pep8 pytest-pep257 pytest-cov pytest-cache pytest-isort
+  - travis_retry pip install --upgrade pytest pytest-pep8 pytest-pep257 pytest-cov pytest-cache pytest-isort
   - travis_retry pip install -e .
```

Comment:
The `pip --upgrade` is perhaps not necessary?  (see also my previous `isort:skip` message)